foundation/okfplugins/feature_block/templates/feature_block_plugin.html

```diff
@@ -75,9 +75,9 @@ <h3 class="title block mb-8 font-bold text-2xl">{{ instance.title }}</h3>
   {% else %}
   </div>
   {% endif %}
-{% elif instance.block_type == 'blue' or instance.block_type == 'purple' %}
-<a href="{{ instance.url }}" class="relative flex items-center justify-center p-4 group aspect-square -has-bg-circle -bg-circle-full {% if instance.block_type != 'blue'%}-bg-circle-okfn-{{ instance.block_type }} {% endif %}}">
-  <h3 class="relative z-10 title block font-bold text-2xl"> {{ instance.title }} </h3>
+{% elif instance.block_type == 'blue' or instance.block_type == 'purple' or instance.block_type == 'yellow'%}
+<a href="{{ instance.url }}" class="content-grid__item-background relative flex items-center justify-center p-4 group aspect-square -has-bg-circle -bg-circle-full {% if instance.block_type != 'blue'%}-bg-circle-okfn-{{ instance.block_type }} {% endif %} text-{{ instance.text_color }}">
```

Comment:
What is the "content-grid__item-background" class? I cannot find it anywhere else in our codebase 🤔

Reply:
NVM, typo in my search.